### PR TITLE
My nc news 9

### PR DESCRIPTION
**Pull request by kamiviolet**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -391,3 +390,32 @@ describe('/api/articles/:article_id/comments', () => {
...
+    it('DELETE - status 400 - invalid comment_id', () => {
+        return request(app)
+            .delete('/api/comments/non-sense')
+            .expect(400)
+            .then(({body}) => {
+                const {message} = body;
+                expect(message).toBe('Invalid request input.')
+            })
+    })
+})
```
Comment (johnhutcheon) [inline __tests__/app.test.js:421]: Nice testing Kami!

Comment (kamiviolet) [reply __tests__/app.test.js:421]: thank you John :D

File: models/comments.model.js [hunk 2]
```diff
@@ -28,4 +30,20 @@ exports.createNewCommentByArticleId = (id, comment) => {
...
+                return Promise.reject(msg);
+            }
+        })
+        .then(() => {
+            return db
+                .query(`
+                    DELETE FROM comments
+                    WHERE comment_id in ($1);
+                `, [id])
+        })
```
Comment (johnhutcheon) [inline models/comments.model.js:48]: Nice!

**Review by johnhutcheon — APPROVED** [review]
Great work Kami, happy for you to merge! 🙂

**Comment by kamiviolet** [comment]
thank you so much! :D

**Pull request merged**